sentry-core/src/test/java/io/sentry/core/transport/HttpTransportTest.kt

```diff
@@ -297,6 +297,66 @@ class HttpTransportTest {
         assertTrue(transport.isRetryAfter("event"))
     }
 
+    @Test
+    fun `parse X-Sentry-Rate-Limit and ignore unknown categories`() {
+        val transport = fixture.getSUT()
+
+        whenever(fixture.connection.inputStream).thenThrow(IOException())
+        whenever(fixture.connection.getHeaderField(eq("X-Sentry-Rate-Limits")))
+            .thenReturn("60:default;wtf;error;security:organization")
```

Comment:
`m`: I'm not sure if keeping `wtf` in our code is a good idea. Can we maybe change it to `foobar`?

Reply:
lol
https://acronyms.thefreedictionary.com/WTF
the meaning is in the eyes of whom read it :)
will do it